cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -77,7 +80,7 @@ public KafkaCruiseControlServlet(AsyncKafkaCruiseControl asynckafkaCruiseControl
                                    long sessionExpiryMs,
                                    MetricRegistry dropwizardMetricRegistry) {
     _asyncKafkaCruiseControl = asynckafkaCruiseControl;
-    _sessionManager = new SessionManager(5, sessionExpiryMs, Time.SYSTEM, dropwizardMetricRegistry, _successfulRequestExecutionTimer);
+    _userTaskManager = new UserTaskManager(sessionExpiryMs, 5, dropwizardMetricRegistry);
```

Comment:
Nit: For ease of identifying the config value, should we move `5` to a static final long `MAX_ACTIVE_USER_TASKS`?